cicd/gitops/gitops-release-notes.adoc

```diff
@@ -32,6 +32,7 @@ ifdef::openshift-enterprise[]
 * xref:../../operators/admin/olm-configuring-proxy-support.adoc#olm-inject-custom-ca_olm-configuring-proxy-support[Injecting a custom CA certificate]
 endif::[]
 
+include::modules/gitops-release-notes-1-8-5.adoc[leveloffset=+1]
```

Comment:
@masauso-lungu Leave a line space between includes to maintain consistency here in this file.